railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -95,14 +95,16 @@
       </dd>
     <% end %>
 
+    <dt>Format:</dt>
     <% if @email.multipart? %>
-      <dt>Format:</dt>
       <dd>
         <select id="part" onchange="refreshBody();">
           <option <%= request.format == Mime[:html] ? 'selected' : '' %> value="<%= part_query('text/html') %>">View as HTML email</option>
           <option <%= request.format == Mime[:text] ? 'selected' : '' %> value="<%= part_query('text/plain') %>">View as plain-text email</option>
         </select>
       </dd>
+    <% else %>
+      <dd id="mime_type" data-mime_type="<%= part_query(@email.mime_type) %>"><%= @email.mime_type == 'text/html' ? 'HTML email' : 'plain-text email' %></dd>
```

Comment:
Since we often use a hyphen for data attributes, can you change it to `data-mime-type`?